plugins/net_plugin/net_plugin.cpp

```diff
@@ -1792,8 +1792,8 @@ namespace eosio {
       }
       // number of blocks syncing node is behind from a peer node
       uint32_t nblk_behind_by_net_latency = static_cast<uint32_t>(network_latency_ns / block_interval_ns);
-      // 2x for time it takes for message to reach back to peer node, +1 to compensate for integer division truncation
-      uint32_t nblk_combined_latency = 2 * nblk_behind_by_net_latency + 1;
+      // 2x for time it takes for message to reach back to peer node
+      uint32_t nblk_combined_latency = 2 * nblk_behind_by_net_latency;
```

Comment:
How about adding a check that if latency is 0, do not  +1; otherwise +1? This would keep the original intention of compensating for integer division truncation.

Reply:
I think the original intention was wrong. I see no reason to round-up when the latency is multiplied by 2 anyway. I think floor is as good if not better than ceiling in this case.